[website] Open 4 new roles (#27123)
mui-org/material-ui: docs/src/pages/company/careers/product-manager.md

- Type of work: Contractor (full-time, convertible to an employee position in the future).
- Start date: Immediately.
- We're a remote company, we prefer asynchronous communication over meetings.
- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.

## About us

See the [careers](/company/careers/) and [about us](https://next.material-ui.com/branding/about/) pages.

### Why we're hiring

Both our open-source community and our premium products are growing fast (x2-3 YoY).
We need talented people to keep that going!

The advanced components team (X) needs help to build its best features.
We have started with the [data grid](/components/data-grid/) a year ago.

### Why this is interesting

Our solution is empowering React developers to build awesome applications. It should be easy, it shouldn't require any advanced technical skills. Hundreds of thousands of developers use Material-UI every month.

Our enterprise components portfolio is still small, with a million interesting and challenging problems to solve.

## About the role

You will be joining the advanced components team.

### What you'll do on a day-to-day basis

> - You will coordinate with the engineering to ensure that the product being delivered at each iteration solves the problem. This involve growing a deep understanding of our technical choices and constraints. If time allows you to, you assit the engineering team, and implement some of the items in the backlog.

Reviewer: ```suggestion
- You will coordinate with the engineering team to ensure that the product being delivered at each iteration solves the problem. This involves growing a deep understanding of our technical choices and constraints. If time allows, you'll assist the engineering team, and implement some of the items in the backlog.
```

- You will drive the revenue and community growth by owning KPIs.
- You will grow and cultivate a deep understanding of the problems that developers have when they deal with enterprise applications. This means that you will observe and reach out to the community, run research interviews and share your insights with the team.
- You will keep a close eye on feature requests, issues, and general improvements (mostly through GitHub issues and emails), to curate opportunities based on our strategy.

> - You will build a strategy for your product area and contributing to the overall product strategy, e.g establishing a go-to-market strategy.

Reviewer: ```suggestion
- You will build a strategy for your product area and contribute to the overall product strategy, e.g establishing a go-to-market strategy.
```

- You will assess the impact of initiatives through telemetry data and qualitative feedback to help us develop our understanding further, and decide on the next steps.

## About you

### Skills you should have

> - +3 years prior experience working as a Front-end engineer with React.

Reviewer: ```suggestion
- +3 years prior experience working with React as a Front-end engineer.
```

- +1 years experience as Product Manager or closely related roles such as Product Owner, Program Manager, or Solutions Architect.

> - Able to switch between the big picture and details view multiple times a day.

Reviewer: ```suggestion
- Able to switch between the big picture and detailed view multiple times a day.
```

### What would be nice if you had, but isn't required

- Experience building developer tools.
- Experience working with open-source, and having interacted with open-source communities.

## Compensation

> A competitive compensation of up to \$140,000 USD/year, depending on the profile. It could go higher for a person that can significantly push the mission forward. You can find the other perks & benefits in the [careers](/company/careers/#perks-amp-benefits) page.

Reviewer: ```suggestion
Competitive compensation of up to \$140,000 USD/year, depending on the profile. It could go higher for a person that can significantly push the mission forward. You can find the other perks & benefits in the [careers](/company/careers/#perks-amp-benefits) page.
```
